### PR TITLE
fix: role update sync grouping policies

### PR DESCRIPTION
**Pull request by tom2nonames**
role update & delete sync grouping policies.

### CODE REVIEW

**Comment by hsluoyz** [comment]
@tom2nonames fix linter:

image

**Comment by hsluoyz** [comment]
@leo220yuyaodog

File: authz/authz.go [hunk 1]
```diff
@@ -112,6 +112,11 @@ p, *, *, *, /cas, *, *
 p, *, *, *, /api/webauthn, *, *
 p, *, *, GET, /api/get-release, *, *
 p, *, *, GET, /api/get-default-application, *, *
+p, *, *, POST, /api/enforce, *, *
+p, *, *, POST, /api/batch-enforce, *, *
+p, *, *, GET, /api/get-all-objects, *, *
+p, *, *, GET, /api/get-all-actions, *, *
+p, *, *, GET, /api/get-all-roles, *, *
```
Comment (leo220yuyaodog) [inline authz/authz.go:119]: Now that the new api is allowed to be called, should we add comments to the Controller's functions to generate the corresponding Api documentation.

**Comment by notdu** [comment]
hi, Do we have any updates on this PR? @tom2nonames @hsluoyz @leo220yuyaodog

**Comment by hsluoyz** [comment]
@notdu the PR author didn't respond to our code reviews for months and now the PR has code conflicts.

If you would like this feature, you can create a new PR (can refer to this PR's code or not), we can continue there

**Comment by notdu** [comment]
ok, i'll open a new PR and refer to this PR.

**Comment by hsluoyz** [comment]
Replaced by: https://github.com/casdoor/casdoor/pull/1477

**Pull request closed without merging**